libvast/test/table_slice.cpp

```diff
@@ -282,7 +282,7 @@ TEST(project column flat index) {
   auto sut = truncate(zeek_conn_log[0], 3);
   auto proj = project(sut, time_type{}, 0, string_type{}, 6);
   CHECK(proj);
-  CHECK_NOT_EQUAL(proj.begin(), proj.end());
+  CHECK(proj.begin() != proj.end());
```

Comment:
Cannot use CHECK_NOT_EQUAL as these require non const operands for new inspector API. begin and end function return const ones